app/components/sidebars/main/main_sidebar.js

```diff
@@ -118,8 +109,8 @@ export default class ChannelDrawer extends Component {
     }
 
     handleAndroidBack = () => {
-        if (this.refs.drawer && this.refs.drawer.isOpened()) {
-            this.refs.drawer.close();
+        if (this.refs.drawer) {
```

Comment:
removing this && part won't be going to end well
you can hook 
onDrawerClose={this.handleDrawerClose}
onDrawerOpen={this.handleDrawerOpen}
to know whether it's open though